packages/media-sound/audacious/audacious.exlib

```diff
@@ -50,6 +51,8 @@ DEFAULT_SRC_CONFIGURE_PARAMS=(
 DEFAULT_SRC_CONFIGURE_OPTION_ENABLES=(
     dbus
     gtk
+    "gtk gtk3"
+    libarchive
```

Comment:
Any reason this doesn't come with a dependency on libarchive?

Reply:
@heirecka Thanks! And since I missed it during review: https://gitlab.exherbo.org/exherbo-unofficial/media-unofficial/-/merge_requests/96